extensions/resteasy-reactive/rest-client-reactive/runtime/src/main/java/io/quarkus/rest/client/reactive/runtime/RestClientBuilderImpl.java

```diff
@@ -65,6 +66,13 @@ public class RestClientBuilderImpl implements RestClientBuilder {
     private String proxyPassword;
     private String nonProxyHosts;
 
+    public RestClientBuilderImpl() {
+        InstanceHandle<ClientLogger> clientLogger = Arc.container().instance(ClientLogger.class);
```

Comment:
This should be done in the `build` method.

Reply:
If this is done in the `build` method, then you have no chance to override this by setting `clientLogger` on builder. As long as there is bean, you can't  set your own logger with builder.

Reply:
No, you are right, you don't have there access to logger anyway.